saleor/core/taxes.py

```diff
@@ -58,8 +55,8 @@ def get_display_price(
 
 
 def quantize_price(
-    price: Union["TaxedMoney", "Money", "Decimal", "TaxedMoneyRange"], currency
-):
+    price: Union["TaxedMoney", "Money", "Decimal", "TaxedMoneyRange"], currency: str
+) -> Union["TaxedMoney", "Money", "Decimal", "TaxedMoneyRange"]:
```

Comment:
We could define a type for this case. What about `T_MoneyLike = Union["TaxedMoney", "Money", "Decimal", "TaxedMoneyRange"]`

Reply:
I dont really like idea of having both Money and Decimal available here. It could lead to strange bugs.

Reply:
I'll modify it in different PR later on, Products module first. Probably I'll handle this util with it too.